Makefile

```diff
@@ -162,7 +162,7 @@ $(FP20COMPILER):
 extract-xiso: $(EXTRACT_XISO)
 $(EXTRACT_XISO):
 	@echo "[ BUILD    ] $@"
-	$(VE)$(MAKE) -C $(NXDK_DIR)/tools/extract-xiso $(QUIET)
+	$(VE)(cd $(NXDK_DIR)/tools/extract-xiso && cmake -G "Unix Makefiles" . $(QUIET) && $(MAKE) $(QUIET))
```

Comment:
Why not `cmake` it in a subdirectory (`build/`)? This would simplify the `distclean`

Reply:
My intent was to keep the path to the extract-xiso binary the same. Probably unimportant, now that I think about it.

Reply:
I changed it.

Reply:
Does "Unix Makefiles" work on msys2?

Reply:
It does, but extract-xiso is currently broken on msys2/mingw. I sumitted https://github.com/XboxDev/extract-xiso/pull/30 to fix it.